CodeStarterCamp_Week1/main.swift

```diff
@@ -8,33 +8,44 @@
 
 import Foundation
 
-// Step 2 (modified)
-
-func makeIcecreamBody(bodywidth: Int, bodyheight: Int) {
-    for _ in 1...bodyheight {
-        for _ in 1...bodywidth {
-            print("*", terminator: "")
-        }
-        print()
-    }
+// Step 3 (1)
+func makeBar(barlength: Int) {
+    for _ in 1...barlength {
+        print(" | |")}}
```

Comment:
이전에도 말씀드렸지만, 중괄호는 일반적으로 개행하시는 편이 좋습니다.
코드 컨밴션을 지키는 것 역시 매우 중요합니다!
```suggestion
        print(" | |")
    }
}
```